src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -474,12 +474,6 @@ private async Task<Either<ActionResult, Unit>> CheckCanDeleteUser(string email)
                 return new ForbidResult();
             }
```

Comment:
You will also need to delete this if statement - github is only letting me highlight the bottom two lines. This will prevent users from being deleted on prod.

`_userDeletionAllowed` basically the variable `enableThemeDeletion` renamed. `enableThemeDeletion` defaults to false in our main IoC / ARM template `infrastructure/templates/template.json`, but is set to be true for dev/test/preprod environments in `infrastructure/parameters/{env}.parameters.json`.

Reply:
Ah I see - is it worth just removing this method entirely and replacing the check with `CheckCanManageAllUsers`?